recipes/pcre/all/CMakeLists.txt

```diff
@@ -4,4 +4,10 @@ project(cmake_wrapper)
 include(conanbuildinfo.cmake)
 conan_basic_setup()
 
+if(MINGW)
+    # Avoid the following link error:
+    # undefined reference to `__strcpy_chk'
+    link_libraries(ssp)
+endif()
```

Comment:
Is it robust? Maybe check existence of ssp, then link?

Reply:
On mingw@Linux
```
find_library(SSP_LIBRARY NAMES ssp)
```
will only work if you have set up [`CMAKE_FIND_ROOT_PATH`](https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_ROOT_PATH.html) correctly using a cmake toolchain file.

But I'm starting to think this is a problem with the mingw toolchain of my linux distribution (Fedora Linux).
The complete error is:
```
/usr/lib/gcc/x86_64-w64-mingw32/10.3.1/../../../../x86_64-w64-mingw32/bin/ld: /usr/x86_64-w64-mingw32/sys-root/mingw/lib/../lib/libmingwex.a(lib64_libmingwex_a-dirent.o):(.text+0x2d6): undefined reference to `__strcpy_chk'
```
So it looks like `libmingwex.a` is compiled with `FORTIFY_SOURCE`. The toolchain should also link to libssp when linking to libmingwex. Not sure though.

Reply:
I've opened a bug report here: https://bugzilla.redhat.com/show_bug.cgi?id=2002656

I'll remove this mingw modification for now.